var/spack/repos/builtin/packages/brion/package.py

```diff
@@ -17,7 +17,8 @@ class Brion(CMakePackage):
 
     version('develop', submodules=True)
     version('3.1.0', tag='3.1.0', submodules=True)
-    version('3.2.0', tag='3.2.0', submodules=True, preferred=True)
+    version('3.2.0', tag='3.2.0', submodules=True)
+    version('3.3.0', tag='3.3.0', submodules=True, preferred=True)
```

Comment:
FWIW, `preferred` is not needed for the highest numeric version, that's the default.